BYT/View Controllers/FoaasViewController.swift

```diff
@@ -66,6 +67,17 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         self.view.backgroundColor = .white
         self.view.addSubview(foaasSettingsMenuView)
         self.view.addSubview(foaasView)
+        self.foaasView.backgroundColor = ColorManager.shared.currentColorScheme.primary
+    }
+    
+    private func updateSettingsMenuColors() {
+        if ColorManager.shared.colorSchemes != nil {
+            DispatchQueue.main.async {
+                self.foaasSettingsMenuView.view1.backgroundColor = ColorManager.shared.colorSchemes[0].primary
+                self.foaasSettingsMenuView.view2.backgroundColor = ColorManager.shared.colorSchemes[1].primary
+                self.foaasSettingsMenuView.view3.backgroundColor = ColorManager.shared.colorSchemes[2].primary
```

Comment:
we can leave this the way it is for now. but since the `colorSchemes` object has a property called `color_scheme_name`, it might be more descriptive if we could somehow retrieve a colorScheme by name rather than array subscript